cgi-bin/LJ/Widget/CustomizeTheme.pm

```diff
@@ -222,7 +222,7 @@ sub render_body {
             propgroup  => "customcss",
             groupprops => $groups{groupprops}->{customcss},
         );
-        $ret .= <<EOF
+        $ret .= <<EOF;
```

Comment:
Does this construction actually work or did Tidy do something weird?

Reply:
Yes. The only reason omitting the semicolon wasn't a syntax error was that the here doc is followed by a close curly brace.

Reply:
Okay cool, I've been jumping between languages so much lately I'm losing track of line termination rules, haha.